src/main/java/it/gov/pagopa/pu/debtpositions/mapper/InstallmentPaidPIIMapper.java

```diff
@@ -0,0 +1,107 @@
+package it.gov.pagopa.pu.debtpositions.mapper;
+
+import it.gov.pagopa.pu.debtpositions.citizen.service.PersonalDataService;
+import it.gov.pagopa.pu.debtpositions.dto.InstallmentPIIDTO;
+import it.gov.pagopa.pu.debtpositions.dto.InstallmentPaidViewDTO;
+import it.gov.pagopa.pu.debtpositions.enums.PaymentOutcomeCode;
+import it.gov.pagopa.pu.debtpositions.enums.PersonEntityType;
+import it.gov.pagopa.pu.debtpositions.enums.UniqueIdentifierType;
+import it.gov.pagopa.pu.debtpositions.model.view.installment.InstallmentPaidViewNoPII;
+import org.springframework.stereotype.Service;
+
+@Service
+public class InstallmentPaidPIIMapper {
+
+  private final PersonalDataService personalDataService;
+  private final PersonMapper personMapper;
+
+  public InstallmentPaidPIIMapper(PersonalDataService personalDataService, PersonMapper personMapper) {
+    this.personalDataService = personalDataService;
+    this.personMapper = personMapper;
+  }
+
+  public InstallmentPaidViewDTO mapToInstallmentPaidViewDTO(Float versionTrack, InstallmentPaidViewNoPII installmentPaidViewNoPII){
```

Comment:
we are not interested in versionTrack when performing the query!
These mappers should be moved on activities when mapping the query result into the DTO representing the CSV row line!